buildenv/jenkins/variables/defaults.yml

```diff
@@ -432,12 +435,15 @@ x86-64_linux_valhalla:
 # Linux Aarch 64bits Compressed Pointers
 #========================================#
 aarch64_linux:
-  extends: ['debuginfo', 'freemarker', 'openjdk_reference_repo', 'openssl']
+  extends: ['boot_jdk_default', 'debuginfo', 'freemarker', 'openjdk_reference_repo', 'openssl']
   boot_jdk:
-    8: '/usr/lib/jvm/java-1.8.0'
-    11: '/usr/lib/jvm/adoptojdk-java-11'
-    15: '/usr/lib/jvm/jdk-15+36'
-    next: '/usr/lib/jvm/jdk-15+36'
+    version:
+      8: '8'
+      11: '11'
+      15: '15'
```

Comment:
How does this work? There isn't any Openj9 Aarch64 build at Adopt for Java 15.

Reply:
It will only try if the directory doesn't already exist on the machine.  So for the odd case where the platform is still in progress that will need to be pushed out manually otherwise it will fail.

Reply:
This PR changes the default bootjdk directories, so none of them exist unless the machines have been updated. 
`${HOME}/bootjdks` for most, and `/opt/bootjdks` for AIX.

Reply:
Correct, for the single aarch machine I have already copied the bootjdk to the new location.
```
[jenkins@cent7-arm8-1 ~]$ ll bootjdks/
total 12
drwxrwxr-x 9 jenkins jenkins 4096 Oct  6 05:06 jdk11
drwxrwxr-x 9 jenkins jenkins 4096 Oct  6 12:22 jdk15
drwxrwxr-x 7 jenkins jenkins 4096 Oct  6 12:25 jdk8
```